pypackage/cookiecutter.json

```diff
@@ -17,6 +17,7 @@
     "__pypi_url": "https://pypi.org/project/{{ cookiecutter.slug }}",
     "_extensions": ["local_extensions.LocalJinja2Extension"],
     "_directory": "pypackage",
+    "__hdev_project_type": "library",
```

Comment:
Existing projects that have already had one of the new cookiecutter applied won't have a `__hdev_project_type` in their `cookiecutter.json`. When that happens the cookiecutter just defaults to the setting from this `cookiecutter.json` file in the cookiecutter itself. But any project can always add an `__hdev_project_type` to its `cookiecutter.json` to override the default